ansible_collections/arista/avd/roles/eos_designs/doc/ptp.md

```diff
@@ -145,28 +150,63 @@ Alternatively the default `node_type_keys` can be overridden to add a `ptp_leaf`
 The automatic PTP priorities can be manually overriden if required, for example for blue-leaf1:
 
 ```yml
-nodes:
-  blue-leaf1:
-    ptp:
-      enabled: true
-      priority1: < 0-255 | default -> automatically set based on node_type >
-      priority2: < 0-255 | default -> (node_id modulus 256) >
+l3leaf:
+  node_groups:
+    leaf1:
+      nodes:
+        leaf1:
+          ptp:
+            enabled: true
+            priority1: < 0-255 | default -> automatically set based on node_type >
+            priority2: < 0-255 | default -> (node_id modulus 256) >
 ```
 
 ### PTP Clock Identity
 
 #### Setting PTP Clock Identity automatically
 
 By default PTP clock identity is generated and set automatically.
+The clock identity will consist of the following:
 
-```yml
-auto_clock_identity = < true | false | default -> true >
-clock_identity = < (clock_identity_prefix = 00:1C:73 (default)) + (PTP priority 1 as HEX) + ":00:" + (PTP priority 2 as HEX) >
-```
+- a prefix which is "00:1C:73" by default
+- PTP priority 1 as HEX
+- ":00:"
+- PTP priority 2 as HEX
+
+This means, that for a node with PTP priority 1 = 30 and PTP priority 2 = 2, the PTP clock identity will automatically be set to: `00:1C:73:1e:00:01`
+
+If you prefer to have PTP clock identity be the system MAC-address of the switch, which is the default EOS behaviour, simply set auto_clock_identity: false on one or more of the levels shown below:
+
+- on the fabric level, for example FABRIC.yml
+
+  ```yml
+  ptp:
+    auto_clock_identity: < true | false | default -> true >
+  ```
+
+- per node_group, for example for all spine nodes in SPINES.yml
+
+  ```yml
+  spine:
+    defaults:
+      ptp:
+        auto_clock_identity: < true | false | default -> true >
+  ```
+
+- per node for a specific node/device inside a node_group, for example for a specific leaf in LEAFS.yml
```

Comment:
Is it necessary to describe this about how to set variables on different levels for _every_ variable? Maybe having a more general article (or section) on variables?

Reply:
Fixed (simplified)